challenges.js

```diff
@@ -9,23 +9,40 @@ const challengeSort = {
 	/**
 	 * Function to read from api. 
 	 */
-	async getApiToArray () {
+	async getApiToArray() {
 		let successful = false;
 		try {
-			const res = await fetch('https://lernia-sjj-assignments.vercel.app/api/challenges');
+			const res = await fetch ('https://lernia-sjj-assignments.vercel.app/api/challenges'); 
 			const data = await res.json();
 			this.ratingArray = data.challenges;
 			successful = true;
 		} catch (error) {
 			console.error('Error in getApiToArray', error);
-		} finally { 
-			if (!successful) { 
+			this.displayErrorMessage('Failed to load challenges')
+		} finally {
+			if (!successful) {
 				this.ratingArray = [];
 			}
 		}
-			
-			return this.ratingArray;
-		},
+
+		return this.ratingArray;
+	},
+
+	displayErrorMessage(message) {
```

Comment:
Maybe we can move displayErrorMessage function above getApiToArray for better readability and updates in the future.